src/lib/jsonParseV2/jsonRequestTreat.cpp

```diff
@@ -79,7 +79,6 @@ std::string jsonRequestTreat(ConnectionInfo* ciP, ParseData* parseDataP, Request
   case EntityAttributeRequest:
     releaseP->attribute = &parseDataP->attr.attribute;
     answer = parseContextAttribute(ciP, &parseDataP->attr.attribute);
-    parseDataP->attr.attribute.present("Parsed Attribute: ", 0);
```

Comment:
I'd suggest to add the folloing to CNR, related with this:

"Fix:  performance problem in NGSIv2 API due to uncontrolled log trace printing"

as I think it is important to note in the 0.24.0 branch CNR this fix.

Of course, this doesn't preclude a new CNR entry about  #1398 once we address it.